.github/workflows/pythonpublish_manual.yml

```diff
@@ -5,25 +5,32 @@ name: Upload Python Package (manual)
 
 on: workflow_dispatch
 
+
 jobs:
   deploy:
-
+    name: Build and Publish to PyPI
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up Python
-      uses: actions/setup-python@v2
+      - name: Checkout Source
+        uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
       with:
         python-version: '3.x'
-    - name: Install dependencies
+
+    - name: Setup build environment 
       run: |
-        python -m pip install --upgrade pip
-        pip install setuptools wheel twine
-    - name: Build and publish
+        python -m pip install --upgrade build twine
+
+    - name: Build source and wheel distributions
+      run: |
+        python -m build
+        twine check --strict dist/*
+
+    - name: Publish to PyPi
       env:
         TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
         TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
-      run: |
-        python setup.py sdist bdist_wheel
-        twine upload --verbose dist/*
+      run: twine upload dist/*
```

Comment:
I think the --verbose flag should be present in the manually triggered workflow, as before.